defaults/aws_config.yaml

```diff
@@ -9,7 +9,7 @@ instance_type_monitor: 't3.large'
 # manual on creating loader AMI's: see docs/new_loader_ami.md
 ami_id_loader: 'scylla-qa-loader-ami-v21-ubuntu22'  # versions: c-s 2024.2.0-rc0, s-b 0.1.21
 # manual on updating monitor images see: docs/monitoring-images.md
-ami_id_monitor: 'scylladb-monitor-4-7-2-2024-05-13t08-38-47z'
```

Comment:
what about the GCP images ?

Reply:
and also updating the `monitor_branch` default, since in some platform we don't have images, i.e. Azure